code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -193,6 +193,12 @@
 	if((src.loc) && isturf(src.loc))
 		if(!stat && !resting && !buckled)
 			for(var/mob/living/simple_animal/mouse/M in view(1,src))
+				if(istype(M, /mob/living/simple_animal/mouse/brown/Tom) && (name == "Jerry")) //Turns out there's no jerry subtype.
+					var/CD

```

Comment:
okay, so you realise the reason I pointed this out is because this doesn't work at all right?

The var is defined at the life proc level, adn will go out of scope immediateyl at the end of it, hence being unset.

You need a class level var for the cat mob.